### PR TITLE
Eventlist

### PR DESCRIPTION
**Pull request by jennydely**
As a event participant
I need a list of events
so that I can see all upcoming events sorted by date

 - It's a long scrolling list of events
 - An event contains an titel with name and location

 EventList with name is sorted by date
events from previews years will not be listed

### CODE REVIEW

File: src/eventPage/Event.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from 'styled-components'
+
+export default function Event({event}) {
+const{eventname, eventlocation, eventdate} = event
```
Comment (sascharissling) [inline src/eventPage/Event.js:6]: Better use camelcase for these destructured variables to increase readability for others
`eventName, eventLocation, eventDate`

File: src/mockDB/events.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        "eventlocation": "Mengede",
+        "eventdate": "11.06.2020 - 14.06.2020"
+    },
+    {
+        "id": "7",
+        "eventname": "WackenWinterNights",
+        "eventlocation": "Wacken",
+        "eventdate": "14.02.2020 - 16.02.2020"
+    }
+]
```
Comment (sascharissling) [inline src/mockDB/events.json:44]: See above. camelcase increases readability :-)

File: src/eventPage/EventList.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import Event from './Event'
+import styled from 'styled-components/macro'
+//import {getEvents} from '../utils/services'
+const jsonEvents = require('../mockDB/events.json')
```
Comment (mpagels) [inline src/eventPage/EventList.js:5]: use ES6 feature "import" instead of "require"

**Review by mpagels — COMMENTED** [review]
Add some little change suggestions

File: src/eventPage/Event.js (hunk 1 above)
Comment (jennydely) [reply src/eventPage/Event.js]: done :)

File: src/mockDB/events.json (hunk 2 above)
Comment (jennydely) [reply src/mockDB/events.json:44]: done

File: src/eventPage/EventList.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import Event from './Event'
+import styled from 'styled-components/macro'
+//import {getEvents} from '../utils/services'
```
[the remark this answers is not in the document]
Comment (jennydely) [reply src/eventPage/EventList.js]: done

File: src/eventPage/EventList.js (hunk 3 above)
Comment (jennydely) [reply src/eventPage/EventList.js]: changed

**Pull request merged**